tests/chainer_tests/graph_optimization_tests/test_static_graph_models.py

```diff
@@ -105,7 +106,9 @@ def setUp(self):
 
     def check_forward(self, x):
         y_dyn = self.chain.dynamic_call(x)
-        with chainer.using_config('enable_backprop', False):
+
+        with chainer.using_config('use_static_graph', self.use_static_graph), \
+                chainer.using_config('enable_backprop', False):
```

Comment:
Let's check if the feature is actually enabled/disabled.

Reply:
Wrote some additional checks.